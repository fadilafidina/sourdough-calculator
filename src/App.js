import React from 'react';
import { Container } from 'react-bootstrap';
import Header from './Header'
import Footer from './Footer'
import Calculator from './Calculator';

function App() {
    return (
        <>
            <Container className="container" aria-label='app'>
                <Header />
                <Calculator />
            </Container>
            <Footer />
        </>
    );
}
export default App;
