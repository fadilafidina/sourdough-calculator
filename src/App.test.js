import { render, screen } from '@testing-library/react'
import React from 'react';
import App from './App'

describe('When render app', () => {
    render(<App />);
    test('should match snapshot', () => {
        expect(screen.getByLabelText('app')).toMatchSnapshot();
    });
});