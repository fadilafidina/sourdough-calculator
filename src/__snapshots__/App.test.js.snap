// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When render app should match snapshot 1`] = `
<div
  aria-label="app"
  class="container container"
>
  <h1>
    Sourdough calculator 
    <span
      aria-label="sparkle"
      role="img"
    >
      âœ¨
    </span>
  </h1>
  <div>
    <form
      class="body"
    >
      <div
        class="pair"
      >
        <div
          class="label"
        >
          Total Flour
        </div>
        <div
          class="left"
        >
          <input
            name="totalFlour"
            placeholder="Ratio of Total Flour"
            type="text"
            value="100"
          />
           %
        </div>
        <div
          class="right"
        >
          Amount:
          <input
            name="totalFlour"
            placeholder="Total Flour"
            type="text"
            value=""
          />
           grams
        </div>
      </div>
      <div
        class="pair"
      >
        <div
          class="label"
        >
          Bread flour
        </div>
        <div
          class="left"
        >
          <input
            name="breadFlour"
            placeholder="Ratio of Bread flour"
            type="text"
            value="80"
          />
           %
        </div>
        <div
          class="right"
        >
          Amount:
          <input
            name="breadFlour"
            placeholder="Bread flour"
            type="text"
            value=""
          />
           grams
        </div>
      </div>
      <div
        class="pair"
      >
        <div
          class="label"
        >
          Whole wheat flour
        </div>
        <div
          class="left"
        >
          <input
            name="wwFlour"
            placeholder="Ratio of Whole wheat flour"
            type="text"
            value="20"
          />
           %
        </div>
        <div
          class="right"
        >
          Amount:
          <input
            name="wwFlour"
            placeholder="Whole wheat flour"
            type="text"
            value=""
          />
           grams
        </div>
      </div>
      <div
        class="pair"
      >
        <div
          class="label"
        >
          Water
        </div>
        <div
          class="left"
        >
          <input
            name="water"
            placeholder="Ratio of Water"
            type="text"
            value="70"
          />
           %
        </div>
        <div
          class="right"
        >
          Amount:
          <input
            name="water"
            placeholder="Water"
            type="text"
            value=""
          />
           grams
        </div>
      </div>
      <div
        class="pair"
      >
        <div
          class="label"
        >
          Levain
        </div>
        <div
          class="left"
        >
          <input
            name="levain"
            placeholder="Ratio of Levain"
            type="text"
            value="20"
          />
           %
        </div>
        <div
          class="right"
        >
          Amount:
          <input
            name="levain"
            placeholder="Levain"
            type="text"
            value=""
          />
           grams
        </div>
      </div>
      <div
        class="pair"
      >
        <div
          class="label"
        >
          Salt
        </div>
        <div
          class="left"
        >
          <input
            name="salt"
            placeholder="Ratio of Salt"
            type="text"
            value="2"
          />
           %
        </div>
        <div
          class="right"
        >
          Amount:
          <input
            name="salt"
            placeholder="Salt"
            type="text"
            value=""
          />
           grams
        </div>
      </div>
      <div
        class="totalMass"
      >
        Total mass: 
        0
         grams
      </div>
      <br />
      <div
        class="wrapper"
      >
        <div>
          <button>
            Reset data
          </button>
        </div>
        <div>
          <button>
            Download csv
          </button>
        </div>
      </div>
    </form>
  </div>
</div>
`;
